charts/camunda-platform/charts/identity/templates/deployment.yaml

```diff
@@ -112,6 +112,20 @@ spec:
             value: {{ .Values.global.identity.auth.optimize.redirectUrl | quote }}
           - name: KEYCLOAK_USERS_0_ROLES_4
             value: "Web Modeler"
+          - name: KEYCLOAK_CLIENTS_0_NAME
```

Comment:
It does work and that's the same we did at [Camunda Platform Docker](https://github.com/camunda/camunda-platform/blob/main/docker-compose.yaml#L167). This will be refactored thought in `8.2` release, since the Identity team introduces a new profile for connectors.

Reply:
did you test it in Kubernetes with test enabled and connectors enabled? :sweat_smile: 
what's in docker-compose is different (the tests are for Helm repo only)

Reply:
Well, I only do `helm install dev ./charts/camunda-platform --namespace camunda-platform-igor --create-namespace` and the tests I rely on PR 🤔

Reply:
Should I run something manually?

Reply:
yeah, the integration tests are external to the charts.
I've checked it, and it's not tested because `readinessProbe` is not enabled (which is the basic test we have before the integration test).

so let's enable `readinessProbe` by default as mentioned and see how it will act in the CI pipeline.

Reply:
that also will ensure that we will not forget it when the connectors component is not enabled by default in 8.2

Reply:
I switched the sort and made the test to be last (`_1`).